governance/votes/Executive Vote - June 16, 2023.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Template - [Executive Vote] Out-of-Schedule Executive Vote - RWA014-A (Coinbase Custody) DAO Resolution - June 16, 2023
+summary: Approving a DAO Resolution pertaining to RWA014-A (Coinbase Custody).
+date: 2023-06-16T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] Out-of-Schedule Executive Vote - RWA014-A (Coinbase Custody) DAO Resolution - June 16, 2023
+
+The Governance Facilitators and dewiz have placed an out-of-schedule executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
+- A DAO Resolution pertaining to RWA014-A (Coinbase Custody) will be approved, as detailed below.
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+
+---
+
+## Proposal Details
+
+### RWA014-A (Coinbase Custody) DAO Resolution
+
+As per this [forum post](INSERT LINK HERE from the Stability Scope Responsible Facilitators, detailing the rationale for this out-of-schedule executive proposal under [MIP24](https://mips.makerdao.com/mips/details/MIP24), approval of this executive proposal will signal Maker Governance's agreement to the following DAO Resolution to James Asset Trust 3, if this executive proposal passes.
```

Comment:
Fairly sure you'll pick it up, but you are also missing a closing parenthesis here, in addition to the link.